app/views/pages/privacy_policy.html.erb

```diff
@@ -12,123 +12,98 @@
-    </p>
-
+    
     <h2 class="govuk-heading-l">What personal data we collect</h2>
 
     <p class="govuk-body">
-      The personal data we collect will depend on whether you’re an applicant.
-    </p>
-
-    <h3 class="govuk-heading-m">
-      What personal data we collect if you’re an applicant
-    </h3>
-
-    <p class="govuk-body">
-      If you’re an applicant using the Register for a national professional qualification service we may collect the following information about you:
+      When you register for an NPQ we may collect the following information about you:
     </p>
 
-    <ul class="govuk-list govuk-list--bullet">
-      <li>your name</li>
-      <li>your phone number</li>
-      <li>your email address</li>
-      <li>your teacher reference number (TRN)</li>
-      <li>your date of birth</li>
-      <li>your National Insurance number</li>
-      <li>where you work</li>
-      <li>what your role is</li>
+    <ul class="govuk-list govuk-list--bullet">   
+     <li>name</li>
+     <li>phone number</li>
+     <li>email address</li>
+     <li>teacher reference number (TRN</li>
+     <li>date of birth</li>
+     <li>National Insurance number</li>
+     <li>where you work</li>
+     <li>what your role is</li>
+     <li>the institution that awarded your qualified teacher status (QTS)</li>
     </ul>
 
     <h2 class="govuk-heading-l">How we use your personal data</h2>
+    
+    <p class="govuk-body">We use your personal data for the following purposes.</p>
 
-    <h3 class="govuk-heading-m">Processing applications</h3>
-
-    <p class="govuk-body">Processing applications includes the following:</p>
-
+    <h3 class="govuk-heading-m">Processing NPQ registrations</h3>
+    
+    <p class="govuk-body">This includes:</p>
+    
     <ul class="govuk-list govuk-list--bullet">
-      <li>sending your application details to teacher training providers</li>
-      <li>validating teacher reference numbers (TRN)</li>
-      <li>working out any funding which teacher training providers are entitled to</li>
+    
+      <li>sending your registration details to NPQ providers</li>
+      <li>validating your teacher reference number (TRN)</li>
+      <li>working out any funding you’re eligible for</li>
+      <li>getting in touch with you and NPQ providers about your NPQ registration</li> 
+      <li>determining NPQ outcomes</li>
+      <li>comparing your data to other data we hold</li>
       <li>getting in touch if there’s a security issue concerning your data</li>
-      <li>getting in touch with candidates and teacher training providers about applications</li>
-      <li>Determining outcomes of teacher training</li>
-      <li>Match, link and analyse your data along with other datasets</li>
-    </ul>
```

Comment:
List needs closing here with an `</ul>` or the subsequent content is indented

![image](https://user-images.githubusercontent.com/128088/203628083-79e3d61e-1ecf-4e8b-a158-97690123b3ec.png)